packages/job-manager/src/batch/batch.spec.ts

```diff
@@ -65,124 +83,59 @@ describe(Batch, () => {
-                    correlationId: 'messageId-1',
-                },
-                {
-                    id: 'job-2',
-                    state: JobTaskState.completed,
-                    result: JobTaskExecutionResult.success,
-                    correlationId: 'messageId-2',
-                },
-            ];
-            let i = 0;
-            const messages = [
-                {
-                    messageId: 'messageId-1',
-                    messageText: '{}',
-                },
-                {
-                    messageId: 'messageId-2',
-                    messageText: '{}',
-                },
-            ];
-
-            taskMock
-                .setup(async o => o.list(jobId1))
-                .returns(async () => {
-                    if (i++ === 0) {
-                        return Promise.resolve(cloudTaskListResultFirst as any);
-                    } else {
-                        return Promise.resolve(<BatchServiceModels.CloudTaskListResult>{});
-                    }
-                })
+            jobMock
+                .setup(async o => o.listNext(items1.odatanextLink, options))
+                .returns(async () => Promise.resolve(<JobListResponse>(<unknown>items2)))
                 .verifiable();
-            taskMock
-                .setup(async o => o.listNext('nextPageLink'))
-                .returns(async () => Promise.resolve(cloudTaskListResultNext as BatchServiceModels.TaskListResponse))
+            jobMock
+                .setup(async o => o.getTaskCounts(items1.items[0].id))
+                .returns(async () => Promise.resolve(<JobGetTaskCountsResponse>(<unknown>{ active: 1, running: 2 })))
+                .verifiable();
+            jobMock
+                .setup(async o => o.getTaskCounts(items2.items[0].id))
+                .returns(async () => Promise.resolve(<JobGetTaskCountsResponse>(<unknown>{ active: 3, running: 5 })))
                 .verifiable();
-            taskMock
-                .setup(async o => o.addCollection(jobId1, It.isAny()))
-                .returns(async () => Promise.resolve({ value: [] } as BatchServiceModels.TaskAddCollectionResponse));
-
-            const jobTask = await batch.createTasks(jobId1, messages);
-            cloudTaskListResultFirst[0].id = jobTask[0].id;
-            cloudTaskListResultNext[0].id = jobTask[1].id;
-            jobTasksExpected[0].id = jobTask[0].id;
-            jobTasksExpected[1].id = jobTask[1].id;
 
-            const tasksActual = await batch.getCreatedTasksState(jobId1);
+            const poolMetricsInfo = await batch.getPoolMetricsInfo();
 
-            expect(tasksActual).toEqual(jobTasksExpected);
-            taskMock.verifyAll();
+            expect(poolMetricsInfo).toEqual(poolMetricsInfoExpected);
+            poolMock.verifyAll();
         });
+    });
 
+    describe('getCreatedTasksState()', () => {
         it('verifies taskAddParameter data', async () => {
```

Comment:
> it('verifies taskAddParameter d [](start = 7, length = 32)

should this be under createTasks describe group?

Reply:
updated